src/app/collective-rewards/utils/handleErrors.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react'
+import { useAlertContext } from '@/app/providers'
+
+export const useHandleErrors = (errors: { error: Error | null; title: string }[]) => {
```

Comment:
❤️

Reply:
I'm not sure about this, because at the moment more than one error cannot be shown at the same time. I think the idea is great and can be implemented when we have a notification-like error management.

Reply:
Yup, I agree